shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -97,7 +105,8 @@ - (void)layoutSubviews {
     layer.contentsScale = screenScale;
     layer.rasterizationScale = screenScale;
     layer.framebufferOnly = flutter::Settings::kSurfaceDataAccessible ? NO : YES;
```

Comment:
drive by comment: shouldn't this be NO always? We set it to NO here: https://github.com/flutter/engine/blob/fdb33e4ee7a018badbd847d9f4cbf4ba3ca49efe/shell/platform/darwin/ios/ios_surface_metal_impeller.mm#L57

Reply:
I don't think so.  `YES` is the preferable setting since it has better performance.  I think I may have added this switch that allows you to turn it to NO so we can do wide gamut tests looking extended range pixels.  Its possible that NO is needed in that layer, but not on this one.  I don't know off the top of my head the relationship between these 2 layers.

Reply:
This needs to be NO because of backdropfilters and platform views.

Reply:
They're the same layer, so you're setting to YES and then its getting blown away. So just set it to NO here.

Reply:
I verified they are the same layer.  This seems messy if we have setup in 2 different places for the layer.  I guess this setting might be meaningful for the skia backend though?  That could explain why the setup is in 2 places.

Reply:
I'm not going to touch it since it could potentially mess up the skia backend and I think touching it would mean cleaning it up even further to unify the setup.